airflow/models/mappedoperator.py

```diff
@@ -408,8 +390,23 @@ def trigger_rule(self) -> TriggerRule:
 
     @trigger_rule.setter
     def trigger_rule(self, value):
-        # required for mypy which complains about overriding writeable attr with read-only property
-        raise ValueError("Cannot set trigger_rule for mapped operator.")
+        self.partial_kwargs["trigger_rule"] = value
+
+    @property
+    def is_setup(self):
+        return self.partial_kwargs.get("is_setup", False)
+
+    @is_setup.setter
+    def is_setup(self, value):
+        self.partial_kwargs["is_setup"] = value
```

Comment:
I feel these setters aren’t really needed… There should be a way to supply those when a MappedOperator is created.